### PR TITLE
Configurable hipFireSpread

### PR DESCRIPTION
**Pull request by Koud-Wind**

### CODE REVIEW

File: src/main/java/com/paneedah/weaponlib/WeaponFireAspect.java [hunk 1]
```diff
@@ -445,19 +445,21 @@ public void serverFire(EntityLivingBase player, ItemStack itemStack, BiFunction<
...
+                hipFireSpread = BalancePackManager.getGlobalHipFireSpread();
+                hipFireSpread *= BalancePackManager.getGroupHipFireSpread(weapon.getConfigurationGroup());
             }
        	
             damage *= damageMultiplier;
             
            // System.out.println(weapon.getName() + " | " + spawnEntityRocketParticles);
 
            WeaponSpawnEntity bullet = new WeaponSpawnEntity(weapon, player.world, player, weapon.getSpawnEntityVelocity(),
-                   weapon.getSpawnEntityGravityVelocity(), BalancePackManager.getInaccuracy(weapon) + (isAimed ? 0.0f : 2.6f), (float) damage, weapon.getSpawnEntityExplosionRadius(),
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/WeaponFireAspect.java:460]: Why set the default at 0 wasn't it 2.6 before?

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/weaponlib/WeaponFireAspect.java [hunk 2]
```diff
@@ -445,19 +445,21 @@ public void serverFire(EntityLivingBase player, ItemStack itemStack, BiFunction<
...
+                   weapon.getSpawnEntityGravityVelocity(), BalancePackManager.getInaccuracy(weapon) + (isAimed ? 0.0f : (float) hipFireSpread), (float) damage, weapon.getSpawnEntityExplosionRadius(),
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/WeaponFireAspect.java:462]: Why set at 7.5, wasn't it at 2.6 before?

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/weaponlib/WeaponFireAspect.java (hunk 2 above)
Comment (Desoroxxx) [reply src/main/java/com/paneedah/weaponlib/WeaponFireAspect.java:462]: Any reasons at all?

**Comment by Koud-Wind** [comment]
image

**Comment by Koud-Wind** [comment]
7.5
image
0.0
image

**Pull request merged**